mkt/latecustomization/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import mkt.constants.carriers
+import mkt.constants.regions
```

Comment:
Doing mkt imports in migrations is dangerous, as it can attempt to load models before they are ready and break the migration later when changes to that model are introduced in the future. I'm curious, did django generate this one or did you alter it by hand ?